backend/tests/Notifo.Domain.Tests/Integrations/Discord/DiscordTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Notifo.Domain.Integrations.Discord;
+public sealed class DiscordTests
+{
+    [Fact]
+    public void Should_get_integration()
+    {
+        var client = CreateClient();
+        Assert.NotNull(client.System);
+    }
+
+    [Fact]
+    public async Task Should_send_simple_message()
```

Comment:
Not sure, what you are testing here. Is it a integration test or unit test? If it is an integration test we should mark it with a trait:

https://github.com/notifo-io/notifo/blob/573c09703c70a68c39f58ea20b07d54d47fbaf35/backend/tests/Notifo.Domain.Tests/Integrations/Mailjet/MailjetTests.cs#L13